src/engraving/libmscore/scorefont.h

```diff
@@ -65,7 +68,6 @@ class ScoreFont
     static ScoreFont* fallbackFont();
     static const char* fallbackTextFont();
 
-    Sym sym(SymId id) const;
     uint symCode(SymId id) const;
     QString toString(SymId id) const;
```

Comment:
master has an extra function in here (fromCode), I know because I put it there! Not sure why it's gone missing...

Reply:
Because I had not rebased yet :)
Now it's there.